scripts/download_all_domains.py

```diff
@@ -0,0 +1,65 @@
+"""Downloads all domain data sets from FAOSTAT website."""
+
+from faostat_data_primap.download import (
+    download_file,
+    get_html_content,
+    get_last_updated_date,
+    unzip_file,
+)
+from faostat_data_primap.helper.definitions import downloaded_data_path, sources
+
+
+def download_all_domains(sources: list[tuple[str]]) -> list[str]:
+    """
```

Comment:
You could add downloading of the metadata available with the data. Having the methodology descriptions next to the data could be quite helpful

Reply:
But It's less important and can be added later.

Reply:
I would be good to keep old version of the methodology description docs though, to see if things have changed. I assume it will not always be updated, so we might want to use checksums and only store new versions of they have changed and symlink to the old version instead. That could quickly rule out methodology changes (if they are not updated yearly)